modules/installation-configuration-parameters.adoc

```diff
@@ -1150,6 +1158,10 @@ at least two zones.
 |The name of the Azure cloud environment that is used to configure the Azure SDK with the appropriate Azure API endpoints. If empty, the default value `AzurePublicCloud` is used.
 |Any valid cloud environment, such as `AzurePublicCloud` or `AzureUSGovernmentCloud`.
 
+|`platform.azure.defaultMachinePlatform.vmNetworkingType`
+|Enables accelerated networking. Accelerated networking enables single root I/O virtualization (SR-IOV) to a VM, improving its networking performance.
+|`Accelerated` or `Basic`. The default value is `Accelerated`.
```

Comment:
`The default value is `Accelerated``
This is based on the compute/controlPlane.platform.azure.type setting. If accelerated networking on vm size is supported, installer will enable accelerated networking by default, otherwise, it is Basic.